README.md

```diff
@@ -335,6 +344,31 @@ BemEntityName.isBemEntityName(entityName); // true
 BemEntityName.isBemEntityName({ block: 'button' }); // false
 ```
 
+Serialization
+-------------
+
+The `BemEntityName` has `toJSON` method to support `JSON.stringify()` behaviour.
+
+To serialization `stringify` instance of `BemEntityName`.
+
+```js
+const BemEntityName = require('@bem/entity-name');
+
+const name = new BemEntityName({ block: 'input', mod: 'available' });
+
+JSON.stringify(name); // {"block":"input","mod":{"name":"available","val":true}}
+```
+
+To deserialization `parse` serialized JSON string and create instance of `BemEntityName`.
```

Comment:
Use `JSON.parse` to deserialize JSON string and crate an instance of `BemEntityName`.